src/encoded/upgrade/genetic_modification.py

```diff
@@ -294,3 +294,12 @@ def genetic_modification_9_10(value, system):
     if 'donor' in value:
         value['introduced_elements_donor'] = value['donor']
         value.pop('donor')
+
+
+@upgrade_step('genetic_modification', '10', '11')
+def genetic_modification_10_11(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-5968
+    notes = value.get('notes', '')
+    if value.get('introduced_elements', '') == 'gRNAs and CRISPR machinery':
```

Comment:
For this to be a safe upgrade that could be completed again after there was new data, the if statement must also check that guide_type is not present.  This will not cause a problem in the current upgrade, but if there was ever a need to go backwards, i would be concerned that you would overwrite the corrected data.

Reply:
Did we discover that everything that we have is sgRNA, or is that just the assumed defualt?

Reply:
sgRNA is just the assumed default, we expect a few pgRNA but need confirmation before patching those (will be addressed in WRAN-2003)

Reply:
sgRNA is just the assumed default, we expect a few pgRNA but need confirmation before patching those (will be addressed in WRAN-2003)